types/coin.go

```diff
@@ -694,3 +694,12 @@ func ParseCoins(coinsStr string) (Coins, error) {
 
 	return newCoins, nil
 }
+
+// ParseCoinsNormalized parse and normalize coins.
```

Comment:
If we remove `parseCoin`, could we improve the docs of this function (take inspiration from parseCoin)?

Reply:
done

Reply:
also changed `TestParseCoins` to test `ParseCoinsNormalized`, changed the test case a little bit, since we support decimal coins now.